apps/ledger-live-mobile/src/screens/SendFunds/07-SendBroadcastError.tsx

```diff
@@ -0,0 +1,132 @@
+  const { t } = useTranslation();
+  const insets = useSafeAreaInsets();
+
+  const { account } = useSelector(accountScreenSelector(route));
+  const currency = account ? getAccountCurrency(account) : null;
+
+  const error = route.params?.error;
+  const helperUrl = error?.url ?? urls.faq;
+  const gotoSupport = useCallback(() => Linking.openURL(helperUrl), [helperUrl]);
+  const exportLogs = useExportLogs();
+
+  const retry = useCallback(() => {
+    track("SendErrorRetry");
+    navigation.goBack();
+  }, [navigation]);
+
+  if (!error) return null;
+
+  return (
+    <SafeAreaView
+      style={{
+        flex: 1,
+        display: "flex",
+        flexDirection: "column",
+        marginBottom: insets.bottom,
+      }}
+    >
+      <ScrollView style={{ flex: 1, marginBottom: 16 }}>
+        <TrackScreen category="SendFunds" name="ValidationError" currencyName={currency?.name} />
+        <GenericErrorView error={error} hasExportLogButton={false} />
+
+        <Collapsible px={16} mt={24} mb={16} title={t("errors.TransactionBroadcastError.needHelp")}>
+          <Flex px={16}>
+            <InformativeBanner
+              title={t("errors.TransactionBroadcastError.helpCenter.title")}
+              description={t("errors.TransactionBroadcastError.helpCenter.desc")}
+            >
+              <InformativeBannerButton Icon={<Support />} onPress={gotoSupport}>
+                <Text fontWeight="semiBold">
+                  {t("errors.TransactionBroadcastError.helpCenter.cta")}
+                </Text>
+              </InformativeBannerButton>
+            </InformativeBanner>
+
+            <InformativeBanner
+              title={t("errors.TransactionBroadcastError.technical.title")}
+              description={error.message}
+            >
+              <Flex flexDirection="row" columnGap={8}>
+                <InformativeBannerButton Icon={<Download />} onPress={exportLogs}>
+                  <Text fontWeight="semiBold">
+                    {t("errors.TransactionBroadcastError.technical.cta")}
+                  </Text>
+                </InformativeBannerButton>
+                <InformativeBannerButton as={CopyButton} text={error.message} />
+              </Flex>
+            </InformativeBanner>
+          </Flex>
+        </Collapsible>
+      </ScrollView>
+
+      <Flex px={16}>
+        <Button type="shade" onPress={retry}>
+          {t("send.validation.button.retry")}
+        </Button>
+      </Flex>
+    </SafeAreaView>
+  );
+}
+
+type InformativeBannerProps = {
+  title: string;
+  description: string;
+  children: React.ReactNode;
+};
+
+function InformativeBanner({ title, description, children }: InformativeBannerProps) {
+  return (
+    <Card
+      p={6}
+      mb={6}
+      borderRadius={8}
+      backgroundColor="opacityDefault.c05"
+      alignItems="flex-start"
+      rowGap={16}
+    >
+      <Text numberOfLines={3} variant="paragraphLineHeight">
```

Comment:
Not sure that's fitting for help center I think the message should be fully visible even if it's more than 3 lines

Reply:
Done!